### PR TITLE
Database kj

### LINKED ISSUE #20: Build out Router functionality for the multiple pages.

### LINKED ISSUE #22: Card Info page
Build a `card info` page that populates with the info of whatever card was clicked on.

### PR DESCRIPTION
**Pull request by kathrynljackson**
## Description

- Adds Database functionality
- Current **Lighthouse** Score: 96%

## Motivation and Context

- Add individual card pages that the user can access when they click on the link in `CardDatabase`.
- Fixed NavBar (links now work)
- Updated the color scheme (all colors listed in `App.css`)
- Removed `Container` element surrounding `NavBar` (made formatting easier for me)
- Added individual keys to each `Link` in `CardDatabase` (accessibility)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Issues

- Closes #22
- Closes #20

### CODE REVIEW

**Review by kathrynljackson — COMMENTED** [review]
We still need to remove unused imports. Meant to do that and totally forgot!

File: src/App/App.js [hunk 1]
```diff
@@ -32,49 +32,26 @@ class App extends Component {
...
-            </Row>
-          </Container>
-          } 
-          } />
+          <Route exact path="/database/:cardId" 
+            render={({ match }) => {
+              const { cardId } = match.params;
+              return <CardPage thisCardId={cardId}/>
+            }} 
+          />
```
Comment (CalebCyphers) [inline src/App/App.js:54]: Love this routing

File: src/CardPage/CardPage.js [hunk 2]
```diff
@@ -3,23 +3,43 @@ import './CardPage.scss';
...
+import { Link } from 'react-router-dom';
 
 class CardPage extends Component {
     constructor(props) {
+        super(props);
         this.state = {
-            card: undefined,
-            id: this.card.id
+            currentCardId: this.props.thisCardId,
+            card:'',
```
Comment (CalebCyphers) [inline src/CardPage/CardPage.js:15]: love the state here

File: src/CardPage/CardPage.js [hunk 3]
```diff
@@ -3,23 +3,43 @@ import './CardPage.scss';
...
+                    <img src={tarot.image} />
+                    <p>{tarot.theme}</p>
+                    <p>{tarot.description}</p>
+                    <Link to='/database'>Back</Link>
+                </section>
+            </main>
+        )
+    }
+
+ }
```
Comment (CalebCyphers) [inline src/CardPage/CardPage.js:42]: Looks good! we'll clean up the styling later

**Review by CalebCyphers — APPROVED** [review]

**Pull request merged**